code/modules/mob/living/carbon/xenomorph/castes/widow/abilities_widow.dm

```diff
@@ -166,6 +166,37 @@
 	spiderlings -= source
 	UnregisterSignal(source, list(COMSIG_MOB_DEATH, COMSIG_PARENT_QDELETING))
 
+// ***************************************
+// *********** Spiderling mark
+// ***************************************
+
+/datum/action/xeno_action/activable/spiderling_mark
+	name = "Spiderling Mark"
+	ability_name = "spiderling_mark"
+	desc = "Send your spawn on a valid target, they will automatically destroy themselves out of sheer fury after 15 seconds."
+	action_icon_state = "spiderling_mark"
+	plasma_cost = 50
+	cooldown_timer = 5 SECONDS
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_SPIDERLING_MARK,
+	)
+
+/datum/action/xeno_action/activable/spiderling_mark/use_ability(atom/A)
+	. = ..()
+	var/datum/action/xeno_action/create_spiderling/create_spiderling_action = owner.actions_by_path[/datum/action/xeno_action/create_spiderling]
+	if(length(create_spiderling_action.spiderlings) <= 0)
+		owner.balloon_alert(owner, "No spiderlings")
+		return fail_activate()
+	if(!isturf(A))
+		SEND_SIGNAL(owner, COMSIG_SPIDERLING_MARK, A)
+		owner.balloon_alert(owner, "Spiderlings attacking " + A.name)
+	else
+		SEND_SIGNAL(owner, COMSIG_SPIDERLING_MARK)
+		owner.balloon_alert(owner, "Nothing to attack")
+		return fail_activate()
```

Comment:
2 places sending the same signal isnt allowed just have A = null for the signal when it's !turf